README.en.md

```diff
@@ -129,6 +129,7 @@ _On right side the change from the last update can be seen (28.9.2019)_
 - [Vue.js Slovenia](https://www.meetup.com/vue-slovenia/) | 433 | :chart_with_upwards_trend: 4
 - [Backend developers of Ljubljana](https://www.meetup.com/Backend-developers-of-Ljubljana/) | 385 | :chart_with_upwards_trend: 98
 - [Blockchain Developers Meetup](https://www.meetup.com/Blockchain-Developers-Ljubljana/) | 375 | :chart_with_upwards_trend: 3
+- [Slovenian Golang User Group](https://www.meetup.com/Slovenian-Go-lang-User-Group) | 263 | :chart_with_upwards_trend: 45
```

Comment:
Currently the list is sorted by number of members, please sort this one to an appropriate position :)

Reply:
Lol sorry :D I thought (in my mind) I ordered by number, but looks like I treated this as 363 :D fixed.